components/org.wso2.carbon.identity.api.server.branding.preference.management/org.wso2.carbon.identity.api.server.branding.preference.management.v1/src/main/resources/branding-preference.yaml

```
@@ -0,0 +1,278 @@
+      responses:
+        '200' :
+          description: Successfully updated
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+    delete:
+      tags:
+        - Branding Preference
+      operationId: deleteBrandingPreference
+      summary: Deletes branding preferences of Tenant.
+      description: |
+        This API provides the capability to delete the branding preferences of the tenant.<br>
+          <b>Permission required:</b> <br>
+            * /permission/admin/manage/identity/applicationmgt/update <br>
+          <b>Scope required:</b> <br>
+            * internal_application_mgt_update
+      parameters:
+        - $ref: '#/components/parameters/typeQueryParam'
+        - $ref: '#/components/parameters/nameQueryParam'
+        - $ref: '#/components/parameters/localeQueryParam'
+      responses:
+        '204' :
+          description: Successfully deleted.
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+components:
+  parameters:
+    typeQueryParam:
+      in: query
+      name: type
+      required: false
+      description: Resource Type to filter the retrieval of themes.
+      schema:
+        type: string
+        enum:
+          - ORG
+          - APP
+          - CUSTOM
+      example: "ORG"
+    nameQueryParam:
+      in: query
+      name: name
+      required: false
+      description: Application name/ Tenant name to filter the retrieval of themes.
+      schema:
+        type: string
+      example: "NAME"
+    localeQueryParam:
+      in: query
+      name: locale
+      required: false
+      description: Resource Localye to filter the retrieval of themes.
+      schema:
+        type: string
+      example: "en-US"
+
+  schemas:
+    BrandingPreferenceModel:
+      required:
+        - type
+        - object
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+            - ORG
+            - APP
+            - CUSTOM
+          example: "ORG"
+        name:
+          type: string
+          example: "NAME"
+        locale:
+          type: string
+          example: "en-US"
+        preference:
+          type: object
+          description: "This is the JSON structured branding preference"
+
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          type: string
+          example: TBM-60001
```

```suggestion
          example: BPM-60001
```
BPM is the prefix defined in constants. Use the same code here

---

Addressed with https://github.com/wso2/identity-api-server/pull/332/commits/6056906c0fed0b3f691f708c082a7e335715ce24